libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/RustMatrixRoom.kt

```diff
@@ -253,26 +253,39 @@ class RustMatrixRoom(
         withContext(roomDispatcher) {
             if (originalEventId != null) {
                 runCatching {
-                    innerRoom.edit(messageEventContentFromParts(body, htmlBody), originalEventId.value, transactionId?.value)
+                    innerRoom.edit(messageEventContentFromParts(body, htmlBody), originalEventId.value)
                 }
             } else {
                 runCatching {
                     transactionId?.let { cancelSend(it) }
-                    innerRoom.send(messageEventContentFromParts(body, htmlBody), genTransactionId())
+                    innerRoom.send(messageEventContentFromParts(body, htmlBody))
                 }
             }
         }
 
+    private var inReplyToEventTimelineItem: EventTimelineItem? = null
```

Comment:
Maybe we should take the opportunity to create a `SendService`, so this doesn't pollute the MatrixRoom interface?

Reply:
Yes, `MatrixRoom` is starting to be a bit big... Later?